.experimenter.yaml

```diff
@@ -82,3 +82,11 @@ unified-search:
     enabled:
       type: boolean
       description: "If true, the feature shows up in the search bar."
+wallpapers:
+  description: A feature that allows user to set wallpapers to the home screen.
+  hasExposure: true
+  exposureDescription: ""
+  variables:
+    onboarding:
+      type: boolean
+      description: "If true, enables the wallpaper onboarding."
```

Comment:
So far the pattern that we have been following is to group all the functionalities that we have 
in a screen into feature, see the [home screen feature](https://github.com/mozilla-mobile/fenix/blob/main/.experimenter.yaml#L13-L20), this allows the product team to experiment and mix and match multiple functionalities of one screen into a single experiment.
@sarah541 's [PR](https://github.com/mozilla-mobile/fenix/pull/26309/files#diff-680637c1a9cdbef47ee7a3333ef917b741c19c3266663c0c760c0cbbc69b1928R36-R52) is introducing a new feature called `onboarding`, I think this one could play nicely if we include it as a new section of it.

Reply:
I do not have an understanding yet of how Nimbus controls feature flags yet. Ultimately, we want to be able to control the wallpaper onboarding separately from the rest of onboarding. As long as that goal is met, I will defer to @Amejia481's expertise in where to place it